src/lib/core/datetime/calendar-locale.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {inject, TestBed, async} from '@angular/core/testing';
+import {CalendarLocale} from './calendar-locale';
+import {DatetimeModule} from './index';
+import {SimpleDate} from './simple-date';
+
+
+describe('DefaultCalendarLocale', () => {
```

Comment:
These tests don't seem to well-capture using the current locale with fallback behavior. Any ideas on how to improve this?